x-pack/plugins/search_playground/public/embedable.tsx

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
nice this is a clean solution to lazy loading, I'll have to steal this for the console too.